report-processor/src/main/java/org/opentestsystem/rdw/reporting/processor/stream/ReportFailingChannelInterceptor.java

```
@@ -53,7 +53,7 @@ public void afterReceiveCompletion(final Message<?> message,
     private void failReport(final Exception ex, final Message<?> message) {
         if (ex == null) return;
 
-        logger.debug("Problem processing report", ex);
+        logger.warn("Problem processing report", ex);
```

It was silently failing here when I was testing. 
@wtritch helped me to see what is going on, thanks.